recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,36 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+LICENSE = "CLOSED"
+
+UBOOT_MACHINE ?= "xenguest_arm64_defconfig"
+
+SRCREV = "${AUTOREV}"
+SRC_URI = "\
+    git://github.com/arminn/u-boot.git;protocol=https;branch=mr_rcar_pvblock; \
+"
+FILES_${PN} += " \
+      /xt/doma \
```

Comment:
This is too broad. If we by accident install something into this folder then it gets accounted by Yocto

Reply:
fixed

Reply:
Can't see the fix in this patch